hook/discover_3dsmax.py

```diff
@@ -57,33 +57,51 @@ def on_launch_3dsmax_pipeline(session, event):
 
     # This is not needed, but we make sure to remove PySide to not override
     # the one in 3dmax.
-    modules_to_remove = ['PySide', 'PySide2']
-    paths_to_remove = ''
-    for m in modules_to_remove:
-        try:
-            module_path = imp.find_module(m)[1]
-            paths_to_remove = module_path
-        except ImportError:
-            pass
-
-
-    pipeline_max_base_data['integration']['env']  = {
-        '3DSMAX_PLUG_IN_PATH.set': max_connect_plugins_path,
+    # modules_to_remove = ['PySide', 'PySide2']
+    # paths_to_remove = ''
+    # for m in modules_to_remove:
+    #     try:
+    #         module_path = importlib.find_module(m)[1]
+    #         paths_to_remove = module_path
+    #     except ImportError:
+    #         pass
```

Comment:
Remove this code if it's not useful anymore.

Reply:
Removed